data_loss.rst

```diff
@@ -473,3 +474,30 @@ The following command can be used to monitor the progress of any ongoing balance
 .. code-block:: console
 
     btrfs balance status /mnt2/pool-name
+
+.. _rockonrestore:
+
+Rock-on Restore
+^^^^^^^^^^^^^^^
+
+When Rock-ons are not visible after data restore it is possible that they are either running silently in the background
```

Comment:
```suggestion
When Rock-ons are not visible after data restore, or pool import, it is possible that they are either running silently in the background
```

Just to emphasise that sometimes we are just re-installing and importing the pool. Which should then be followed by a config restore, but there might not be one and as you intimate if the Rock-ons-root has been re-activated we could have ghost Rock-ons, or at least ghost docker images running after a by-hand rock-ons-root re-config as-was.